client-js/app/elements/labrad-registry.html

```diff
@@ -230,34 +233,30 @@
     <!-- Dialogs for various registry operations -->
 
     <paper-dialog id="newKeyDialog" modal>
-      <h1>Create new key</h1>
-      <paper-dialog-scrollable>
-        <form>
-          <paper-input id="newKeyInput" label="Key"></paper-input>
-          <paper-textarea id="newValueInput" label="Value"></paper-textarea>
-        </form>
-      </paper-dialog-scrollable>
-      <div class="buttons"> 
+      <h1>Create New Key</h1>
+      <form>
+        <paper-input id="newKeyInput" label="Key"></paper-input>
+        <paper-textarea id="newValueInput" label="Value"></paper-textarea>
+      </form>
+      <div class="buttons">
```

Comment:
The idea previously was to keep the buttons visible in the dialog, even if the text has to scroll. There may have been some bugs with that which need fixing, but I think that's nicer than having the buttons scroll offscreen when the text overflows.